spec/GraphQLOverHTTP.md

```diff
@@ -1,57 +1,82 @@
-You can find our community in the [graphql-over-http channel](https://graphql.slack.com/archives/CRTKLUZRT) on the [GraphQL Foundation slack](https://slack.graphql.org).
+Note: **Stage 0: Preliminary** This spec is under active development, and not
+ready for implementations yet. For more information, please see the
+[Roadmap](https://github.com/graphql/graphql-over-http/blob/master/ROADMAP.md)
+or
+[how to get involved](https://github.com/graphql/graphql-over-http/blob/master/INTERESTED_DEVELOPERS.md).
+You can find our community in the #graphql-over-http channel on the
+[GraphQL Foundation Discord](https://discord.graphql.org).
 
 ---
 
 # GraphQL over HTTP
 
 **Introduction**
 
-HTTP is the most common choice as the client-server protocol when using GraphQL because of its ubiquity.
-However the [GraphQL specification](https://graphql.github.io/graphql-spec/) deliberately does not specify the transport layer.
-
-The closest thing to an official specification is the article [Serving over HTTP](https://graphql.org/learn/serving-over-http/).
-Leading implementations on both client and server have mostly upheld those best practices and thus established
-a de-facto standard that is commonly used throughout the ecosystem.
-
-This specification is intended to fill this gap by specifying how GraphQL should be served over HTTP.
-The main intention of this specification is to provide interoperability between different client libraries, tools
-and server implementations.
+This specification details how libraries that wish to publish and consume
+GraphQL services over HTTP should do so in order to maximize interoperability
+between client libraries, tools, and server implementations. This specification
+does not override or replace the
+[GraphQL specification](https://spec.graphql.org/), it extends it to cover the
+topic of serving GraphQL services over HTTP. If any statement or algorithm in
+this specification appears to conflict with the GraphQL specification, the
+behavior detailed in the GraphQL specification should be used.
+
+The [GraphQL specification](https://spec.graphql.org/) deliberately does not
+specify the transport layer, however HTTP is the most common choice when serving
+GraphQL to remote clients due to its ubiquity.
+
+Previous to this specification, the article
+[Serving over HTTP](https://graphql.org/learn/serving-over-http/)
+([WayBack Machine entry, 1st June 2022](https://web.archive.org/web/20220601155421/https://graphql.org/learn/serving-over-http/))
+on the graphql.org website served as guidance, and leading implementations on
+both client and server have mostly upheld those best practices and thus
+established a de-facto standard that is commonly used throughout the ecosystem.
+This specification aims to codify and expand on this work.
 
 **Copyright notice**
 
-**TBD**
+Copyright © 2022-present, GraphQL contributors
+
+THESE MATERIALS ARE PROVIDED “AS IS.” The parties expressly disclaim any
```

Comment:
```suggestion
THESE MATERIALS ARE PROVIDED “AS IS”. The parties expressly disclaim any
```

Reply:
Not necessarily. I checked the MIT license - which I believe inspired this - and it puts a comma behind the quotes. https://opensource.org/licenses/MIT

Reply:
This copyright notice is copied and pasted from the GraphQL spec; if we are to change it, we should change it there before reflecting it elsewhere:

https://github.com/graphql/graphql-spec/blob/4de87822d2bc807b746c6a37f6a99bb54d705185/spec/GraphQL.md?plain=1#L35

But although the grammar is incorrect, I agree that we should use the standard license, not the one with correct grammar :+1: 

I'm going to close this conversation, but please feel free to raise a PR against the GraphQL spec and we can discuss it there.